backend/substrapp/tests/views/tests_views_algo.py

```diff
@@ -218,6 +222,7 @@ def test_algo_create(self):
             mcreate.return_value = {}
             response = self.client.post(url, data=data, format='multipart', **self.extra)
 
+        assert response.data['metadata'] is not None
```

Comment:
Why not test that it is equal to the actual value ?

Reply:
I just checked, we cannot get the metadata's value by using the response object. I'll remove this test for now.